test/functional/ex_cmds/cd_spec.lua

```diff
@@ -8,8 +8,7 @@ local call = helpers.call
 local clear = helpers.clear
 local command = helpers.command
 local exc_exec = helpers.exc_exec
-
-if helpers.pending_win32(pending) then return end
+local pathsep = helpers.get_pathsep()
```

Comment:
`get_pathsep` depends on an existing session running (`clear()` must be called before using it).

I think we should change `get_pathsep()` to use `package.config:sub(1,1)` , similar to `iswin()` and `pathroot()`.

The only reason I had avoided it up to now, is because I thought maybe `get_pathsep()` was used in situations where 'shellslash' might have changed. I just checked, and all uses of `get_pathsep()` do not depend on 'shellslash'.